packages/nota-client/.npmrc

```diff
@@ -0,0 +1,2 @@
+unsafe-perm = true
+legacy-peer-deps = true
```

Comment:
need to add to keep old npm behaviour
npm 7 new peer dependency behaviour breaks for many react related packages